SampleProjects/TestSomething/test/isr_declaration.cpp

```diff
@@ -0,0 +1,8 @@
+#include <ArduinoUnitTests.h>
+#include <Arduino.h>
+
+// just check if declaration compiles
+ISR (WDT_vect) {
```

Comment:
Please move this to `interrupts.cpp`, and add a comment with some explanation of `WDT_vect` and a link to the `ISR()` docs

Reply:
👍